docs/gettingstarted.md

```diff
@@ -173,6 +219,20 @@ procname were both passed in as command line parameters. If both of
 these parameters are passed in then default configuration settings will
 be used.
 
+The parameters following procname set the following:
+
+  |Parameter|                          Description|
+  |---------------------------------| ----------------------------|
+  |U|                                  Authentication|
+  |localtime|                          localtime or UTC|
+  |g|                                  Garbage collection|
+  |T|                                  Timeout|
+  |w|                                  Memory limit|
+  |load|                               Path to load process file|
+  |startwithall|                       Start using torq.sh| 
+  |extras|                             Add arguments to start up line|
+  |qcmd|                               Command to start q session|

```

Comment:
WE need a bit more detail on each of these.  e.g. 

start with all is to determnine if this process gets started when all is specified
qcmd allows different versions of q to be used, or different command line options e.g. rlwrap, numactl
U/g/T/w are the standard q commnd line args
extras allows you to specify any addition parameters